cpp/dolfinx/fem/Form.h

```diff
@@ -214,67 +167,322 @@ class Form
     return _function_spaces;
   }
 
-  /// Register the function for 'tabulate_tensor' for cell integral i
-  void set_tabulate_tensor(
-      IntegralType type, int i,
-      const std::function<void(T*, const T*, const T*, const double*,
-                               const int*, const std::uint8_t*,
-                               const std::uint32_t)>& fn)
+  /// Get the function for 'kernel' for integral i of given
+  /// type
+  /// @param[in] type Integral type
+  /// @param[in] i Integral number
```

Comment:
Do you mean the integral ID here? I.e. the ID of the domain the integral is defined on.

Reply:
Yes, docs fixed.